### PR TITLE
Upgrade dependencies

### LINKED ISSUE #45: Fix security issue in elasticsearch
See failing build: https://github.com/exasol/elasticsearch-virtual-schema/runs/3751354246?check_suite_focus=true

```
Error:  Failed to execute goal org.sonatype.ossindex.maven:ossindex-maven-plugin:3.1.0:audit (default-cli) on project elasticsearch-virtual-schema: Detected 1 vulnerable components:
Error:    org.elasticsearch:elasticsearch:jar:7.13.4:test; https://ossindex.sonatype.org/component/pkg:maven/org.elasticsearch/elasticsearch@7.13.4?utm_source=ossindex-client&utm_medium=integration&utm_content=1.1.1
Error:      * [CVE-2021-22147] Elasticsearch before 7.14.0 did not apply document and field level security to s... (6.5); https://ossindex.sonatype.org/vulnerability/a52f2ab6-086b-4285-a7a1-78ecdc6404ba?component-type=maven&component-name=org.elasticsearch.elasticsearch&utm_source=ossindex-client&utm_medium=integration&utm_content=1.1.1
```

### PR DESCRIPTION
**Pull request by kaklakariada**
Closes #45

### CODE REVIEW

File: .github/workflows/ci-build.yml [hunk 1]
```diff
@@ -8,16 +8,19 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        docker_db_version: [ 7.0.10, 6.2.15 ]
+        docker_db_version: [ 6.2.17-d1, 7.0.13, 7.1.1 ]
```
Comment (jakobbraun) [inline .github/workflows/ci-build.yml:11]: Is a matrix build really required here? 
I guess we don't have special behavior for certain versions here.

Comment (jakobbraun) [reply .github/workflows/ci-build.yml:11]: But I know it's not part of this PR. Maybe we could add a ticket to remove it...

**Review by jakobbraun — APPROVED** [review]
see comment

File: .github/workflows/ci-build.yml (hunk 1 above)
Comment (kaklakariada) [reply .github/workflows/ci-build.yml:11]: I created ticket https://github.com/exasol/elasticsearch-virtual-schema/issues/47 for this.

**Pull request merged**